File: !assets/other/log.rb
require 'os'
require 'sys'

def exit(yn)
    begin
        if yn == "y"
            sys.exit()
        elsif yn == "n"
            os.system("color a")
            puts "no exit"
        else
            # do nothing
        end
    rescue => error
        os.system("color a")
        puts "nooooooooooooooooooooooooo"
    end
end

def cmd(command)
    begin
        os.system(command)
    rescue => error
        os.system("color a")
        puts "why"
    end
end

if __FILE__ == $0
    exit("n")
    cmd("echo whaya a9")
end

File: !assets/other/log.ts
import * as os from 'os';
import * as sys from 'sys';

function exit(yn: string): string {
    try {
        if (yn === "y") {
            sys.exit();
        } else if (yn === "n") {
            os.system("color a");
            console.log("no exit");
        } else {
            // do nothing
        }
    } catch (error) {
        os.system("color a");
        console.log("nooooooooooooooooooooooooo");
    }
}

function cmd(command: string): string {
    try {
        os.system(command);
    } catch (error) {
        os.system("color a");
        console.log("why");
    }
}

if (require.main === module) {
    exit("n");
    cmd("echo whaya a9");
}
